packages/website/src/components/Modals/ModalForm.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { SubmitHandler, useForm } from 'react-hook-form';
+
+import { Button } from '@coderscamp/ui/components/Button';
+import { Flex } from '@coderscamp/ui/components/Flex';
+import { FormControl } from '@coderscamp/ui/components/FormControl';
+import { HelperText } from '@coderscamp/ui/components/HelperText';
+import { Input } from '@coderscamp/ui/components/Input';
+import { useBreakpointValue } from '@coderscamp/ui/hooks/useBreakpointValue';
+
+import { emailValidator } from '../Contact/Form';
+
+type ModalFormValues = {
+  name: string;
+  email: string;
+};
+
+const requiredValidator = {
```

Comment:
💭 You can reuse the `requiredValidator` like you did with the `emailValidator`

Reply:
I did not export it because in issue the required text for the validator in the modal was "To pole nie może być puste". In the contact form we have "To pole jest wymagane". For consistency, I can export this validator and then we will have the same error information in the modal and the form.

Reply:
Great idea - re-export it so it would be easier for us to be consistent about this message